OpenSim/Common/Logger.cpp

```diff
@@ -150,6 +149,18 @@ void Logger::addFileSink(const std::string& filepath) {
              "removeFileSink() first.", m_filesink->filename());
```

Comment:
@aymanhab thanks for adding the quotes below. You mentioned that "file sink" is not clear to users; perhaps the usage of "file sink" in this warning should be changed to "log file" for consistency.